wqflask/wqflask/templates/search_result_page.html

```diff
@@ -184,6 +184,16 @@
         }
 
         $(document).ready( function () {
+
+            var getParams = function(url) {
+                var parser = document.createElement('a');
+                parser.href = url;
+                var params = parser.search.substring(1);
```

Comment:
**issue**: Please replace `var` with `let`; It enforces lexical scope. We'd get less surprises if somehow in future someone decides to to use the above param names.